test/spec/modules/akamaiDapRtdProvider_spec.js

```diff
@@ -135,40 +188,54 @@ describe('akamaiDapRtdProvider', function() {
 
   describe('dapTokenize and dapMembership incorrect params', function () {
     it('Onerror and config are null', function () {
-      expect(dapUtils.dapTokenize(null, 'identity', null, null)).to.be.equal(undefined);
-      expect(dapUtils.dapMembership(null, 'identity', null, null)).to.be.equal(undefined);
+      expect(dapUtils.dapTokenize(null, 'identity', onDone, null, null)).to.be.equal(undefined);
+      expect(dapUtils.dapMembership(null, 'identity', onDone, null, null)).to.be.equal(undefined);
+      expect(dapUtils.dapEncryptedMembership(null, 'identity', onDone, null, null)).to.be.equal(undefined);
       const config = {
         'api_hostname': 'prebid.dap.akadns.net',
         'api_version': 1,
         'domain': '',
         'segtax': 503
       };
+      const encConfig = {
+        'api_hostname': 'prebid.dap.akadns.net',
+        'api_version': 1,
+        'domain': '',
+        'segtax': 504
+      };
       let identity = {
         type: 'dap-signature:1.0.0'
       };
-      expect(dapUtils.dapTokenize(config, identity, null, null)).to.be.equal(undefined);
-      expect(dapUtils.dapMembership(config, 'token', null, null)).to.be.equal(undefined);
+      expect(dapUtils.dapTokenize(config, identity, onDone, null, null)).to.be.equal(undefined);
+      expect(dapUtils.dapMembership(config, 'token', onDone, null, null)).to.be.equal(undefined);
+      expect(dapUtils.dapEncryptedMembership(encConfig, 'token', onDone, null, null)).to.be.equal(undefined);
     });
+  });
 
-    it('dapGetToken success', function () {
-      let dapTokenizeStub = sinon.stub(dapUtils, 'dapTokenize').returns(onSuccess);
-      expect(dapUtils.dapGetToken(sampleConfig, 'token',
-        function(token, status, xhr) {
-        },
-        function(xhr, status, error) {
-        }
-      )).to.be.equal(null);
-      dapTokenizeStub.restore();
+  describe('Getting dapTokenize, dapMembership and dapEncryptedMembership from localstorage', function () {
```

Comment:
can you write a test that shows these fail when you don't have consent in the us or when gdpr enforcement module is installed (since you appear not to be on the gvl)

Reply:
Hi @patmmccann I have made the changes for listening to the consent and not initializing the module if the consent is not there. Have added the test cases as well for that as suggested. Please let me know if anything else is needed.

Reply:
Have added the details in the docs repo under this commit : https://github.com/prebid/prebid.github.io/pull/3700/commits/42c1e9e489542b8724f9d0b8bad137522cd42248